.azure-pipelines/testscripts_analyse/constant.py

```diff
@@ -19,5 +19,9 @@
         "multi-asic-t1-lag": "t1",
         "t2": "t2",
         "wan-pub": "wan",
-        "dpu": "dpu"
+        "dpu": "dpu",
+        "tgen": "tgen",
+        "multidut-tgen": "tgen",
+        "snappi": "snappi",
+        "ptf": "ptf"
```

Comment:
Please make sure,the scripts belong to these new added topo type will be added in skip list. Otherwise, it will affect the accurancy of our total caculation.

Reply:
Sure, will add to skip list